README.md

```diff
@@ -1,207 +1,237 @@
-and adding its path to your JVM startup options:
+The SignalFx Java Agent is a
+[Java Virtual Machine](https://docs.oracle.com/javase/7/docs/api/java/lang/instrument/package-summary.html)
+(JVM) agent that automatically instruments your Java application to capture and
+report distributed traces to SignalFx. Download the JAR for the agent's 
+[latest release](https://github.com/signalfx/signalfx-java-tracing/releases/latest)
+and add its path to your JVM startup options:
 
 ```bash
-$ # Specify the Java Agent with the appropriate option and launch your application as usual
-$ java -javaagent:signalfx-tracing.jar -jar app.jar
+$ java -javaagent:path/to/signalfx-tracing.jar -jar app.jar
 ```
+For more information, see [Configure the SignalFx Java Agent](#Configure-the-SignalFx-Java-Agent).
 
-In addition to using bytecode manipulation to instrument supported libraries and
-frameworks, the SignalFx Java Agent configures an OpenTracing-compatible tracer
-to capture and export trace spans. It registers this tracer as the OpenTracing
-`GlobalTracer` to easily enable custom instrumentation throughout your
-application:
+The agent instruments supported libraries and frameworks with bytecode
+manipulation and configures an OpenTracing-compatible tracer to capture
+and export trace spans. The agent also registers the tracer as the OpenTracing
+`GlobalTracer` so you can add custom instrumentation to your application later.
 
-```java
-import io.opentracing.util.GlobalTracer;
+By default, the tracer has constant sampling (i.e., 100% chance of tracing) and
+reports every span. Where applicable, context propagation uses
+[B3 headers](https://github.com/openzipkin/b3-propagation).
 
-public class MyClass {
-  public void MyLogic() {
-    // Use the GlobalTracer utility to create and modify an active span.
-    final Tracer tracer = GlobalTracer.get();
+For more information about configuring and using the agent, see
+the [examples](https://github.com/signalfx/tracing-examples/tree/master/signalfx-tracing/signalfx-java-tracing).
 
-    // Depending on MyLogic's context within a supported framework or library, the custom span
-    // created here can automatically become part of an applicable agent-generated trace,
-    // like those created for web framework filter activity.
-    final Span span = tracer.buildSpan("MyOperation").start();
-    try (Scope scope = tracer.scopeManager().activate(span)) {
-        span.setTag("MyTag", "MyValue");
-        <your functionality...>
-    } finally {
-       span.finish();
-    }
-  }
-}
-```
+## Requirements and supported software
 
-## Supported Runtimes and Languages
+These are the requirements and supported software for the agent.
 
-**The SignalFx Java Agent is designed to work with Java runtimes version 8 or
-above.**
+Specify the SignalFx Java Agent as the only JVM agent for your application.
+If you specify multiple agents, you may encounter issues with at least one
+of them.
```

Comment:
This is a repeat of above -- remove. In addition, the warning should be above.

Reply:
I don't follow. What should we remove?